pkg/apis/vpc/v1alpha1/subnetconnectionbindingmap_types.go

```diff
@@ -0,0 +1,61 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +kubebuilder:validation:XValidation:rule="has(self.targetSubnetSetName) && !has(self.targetSubnetName) || !has(self.targetSubnetSetName) && has(self.targetSubnetName)",message="Only one of targetSubnetSetName or targetSubnetName can be specified"
+type SubnetConnectionBindingMapSpec struct {
+	// SubnetName is the Subnet name which this SubnetConnectionBindingMap is associated.
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="subnetName is immutable"
+	SubnetName string `json:"subnetName"`
+	// TargetSubnetSetName specifies the target SubnetSet which a Subnet is bound to.
```

Comment:
Should be "bound to" or "connected too"?

Reply:
Updated with "connected to"